src/main/resources/data/minecolonies/tags/items/breakable_ore.json

```diff
@@ -0,0 +1,23 @@
+{
+    "replace": false,
+    "values": [
+      "minecraft:coal_ore",
+      "minecraft:deepslate_coal_ore",
+      "minecraft:iron_ore",
+      "minecraft:deepslate_iron_ore",
+      "minecraft:copper_ore",
+      "minecraft:deepslate_copper_ore",
+      "minecraft:gold_ore",
+      "minecraft:deepslate_gold_ore",
+      "minecraft:redstone_ore",
+      "minecraft:deepslate_redstone_ore",
+      "minecraft:emerald_ore",
+      "minecraft:deepslate_emerald_ore",
+      "minecraft:lapis_ore",
+      "minecraft:deepslate_lapis_ore",
+      "minecraft:diamond_ore",
+      "minecraft:deepslate_diamond_ore",
+      "minecraft:nether_gold_ore",
+      "minecraft:nether_quartz_ore"
```

Comment:
These should use the minecraft ore tags (e.g. `#minecraft:coal_ores` includes both regular and deepslate already).

And preferably, there should be a datagen for this rather than being hand-written (although this is not without precedent).

But also if this is going to be a tag for ore blocks then it should be used _instead of_ `Tags.Items.ORES`, not in addition to it.  Or rather, this tag should just include the standard ores tag plus (temporarily) the vanilla ores that Forge hasn't added yet, such that additional modded ores will likely already be included in the list without need for further datapack adjustment.  Either way, the code should not be checking both.

Reply:
Yeah, joining the vanilla ore tag + the missing here would be better

Reply:
Might need a safety check in the code such that if the result of breaking a `breakable_ore` is still a `breakable_ore` then it doesn't get stuck (just ignores it for breaking, and either smelts if smeltable or ignores entirely until the couriers remove), since some modded ores may still drop as themselves instead of as raw ores.  But I don't think the tag itself is the right way to distinguish that.

Reply:
Not using the ores tag was intentional, because of modded ores.

Reply:
Ohh, these here are the breakable ores, yes, we shouldn't just have a generic ore tag in here. We could include though things like #minecraft:coal_ores because we know those are broken down properly.

Reply:
I wasn't sure which of those were right, and which weren't, and was having troubles viewing them so opted for consistency.